pages/vendors.tsx

```diff
@@ -35,19 +37,89 @@ function Vendors(props: VendorProps): JSX.Element {
         return <Error title="Oops, small mistake here..." statusCode={502} />;
     }
 
+    function generateAccumArray(len) {
+        let arr = []
+        for(let i = 0; i < len; i++) {
+            arr.push([])
+        }
+        return arr
+    }
+
+    function getLabelByContinentCode(code) {
+        let label = ''
+        const [continent, country] = code.split('-')
+
+        countryIsoList.forEach(countryIso => {
+            if (continent === 'ALL') {
+                label = 'Worldwide'
+            }
+            if (countryIso.continentCode === continent) {
+                label = countryIso.continentName
+            }
+        });
+        return label
```

Comment:
Kan je deze functioneel opzetten? Dit is vrij imperatief. 
Je kan bijv. Een label "gewoon" aan sorted vendors toevoegen zodat je in 110 niet meer een functie hoeft aan te roepen maar 'label'.